src/vorta/log.py

```diff
@@ -14,6 +14,16 @@
 logger = logging.getLogger()
 
 
+class LogTimedRotatingFileHandler(TimedRotatingFileHandler):
+    """Create subclass of TimedRotatingFileHandler that always appends '.log' to log files."""
+
+    def __init__(self, filename, when='d', interval=1, backupCount=5, encoding=None, delay=False, utc=False):
+        super().__init__(
+            filename, when=when, interval=interval, backupCount=backupCount, encoding=encoding, delay=delay, utc=utc
+        )
+        self.suffix += '.log'
```

Comment:
so you append .log to whatever is already there.

what exactly is already there at init time? does that vary?

Reply:
https://github.com/python/cpython/blob/main/Lib/logging/handlers.py#L431-L432

This shows how the suffix is ueed. The suffix is constructed during init:

https://github.com/python/cpython/blob/b378d991f8cd41c33416e590cb83472cce1d6b98/Lib/logging/handlers.py#L210

While the solution I initially came up with and which was implemented in this PR works, I noticed that there is a documented way we should use instead:

https://docs.python.org/3.12/library/logging.handlers.html#logging.handlers.BaseRotatingHandler.namer

Reply:
@ThomasWaldmann At init time the suffix is set to a year-month-day timestamp [here](https://github.com/python/cpython/blob/cb88ae635e96d7020ba6187bcfd45ace4dcd8395/Lib/logging/handlers.py#LL246C20-L246C20). This suffix then gets appended [here](https://github.com/python/cpython/blob/main/Lib/logging/handlers.py#L431-L432) to whichever filename (in this case, "vorta.log") is passed to the `TimedRotatingFileHandler`. I see now that editing the suffix directly may not be the best approach.

@real-yfprojects After reviewing the code and documentation you linked, maybe a better approach would be to create a function for `namer`. Currently the log files are getting named "vorta.log.[timestamp]". I was thinking of making a function that names them "vorta.[timestamp].log" and setting namer to this function. What do you think of this approach?

Reply:
If you set `when="S"`, you could do a realistic test in a few seconds (instead of days).

Reply:
Definitely! I've been doing that to test my local changes.